### PR TITLE
N16: F1 Telemetry TUI

### PR DESCRIPTION
**Pull request by aldidana**
_Part of #365 ("Newsletter 16")_

### CODE REVIEW

File: content/posts/newsletter-016/index.md [hunk 1]
```diff
@@ -67,6 +67,18 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library & Tooling Updates
 
+### [F1 Telemetry TUI]
+
+![f1-telemetry-tui](https://user-images.githubusercontent.com/6572635/97776468-db1daa00-1b9a-11eb-8d4b-af0b540a9930.gif)
+
+[F1 Telemetry TUI] by [@aldidana] is a Terminal Telemetry for F1 Video Games
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-016/index.md:74]:
```suggestion
[F1 Telemetry TUI] by [@aldidana] is a terminal telemetry tool for F1 video games.
```

**Review by 17cupsofcoffee — APPROVED** [review]
Looks good, few suggestions for formatting changes but nothing worth blocking the PR on.

(also, as an F1 fan, this tool looks really cool 😍)

**Review by 17cupsofcoffee — APPROVED** [review]

**Review by ozkriff — APPROVED** [review]
(Embedded the GIF)

**Comment by ozkriff** [comment]
Thanks!

**Pull request merged**